cpu/native/include/native_internal.h

```diff
@@ -64,6 +64,13 @@ typedef void (*_native_callback_t)(void);
  * @cond INTERNAL
  * internal functions
  */
+/**
+ * @brief   Don't use Linux' `sighandler_t` as it is not POSIX compliant
+ *
+ * @see e.g. https://www.freebsd.org/cgi/man.cgi?query=signal&section=3
+ */
+typedef void (*real_sig_t) (int);
```

Comment:
not needed anymore, right?

Reply:
Oh yeah, sorry.... still on my first coffee ^^"

Reply:
enjoy that first ☕️